client/src/components/List.jsx

```diff
@@ -56,17 +30,39 @@ class List extends Component {
 					<input type="submit" value="Submit" />
 				</form>
 
-				{(this.state.sublist || []).map((item) => (
+				{(this.props.sublist || []).map((item) => (
 					<ListItem id={item.id} title={item.listItem} />
 				))}
 			</div>
 		);
 	}
 }
 
-function mapStateToProps(state) {
+// TODO: derive list state from store state and ownProps here.
+function mapStateToProps(state, ownProps) {
+	const { id } = ownProps.match.params
 	const { lists } = state;
-	return { lists };
+
+	const list = lists.filter(list => {
+		console.log(list)
+		return list.id === id
+	})
+
+	console.log(list)
```

Comment:
As we found during the call, something wasn't working quite right here. An issue with doing things under a time pressure 😆 

The purpose of what we spoke about here was to transform the data from the Redux store, and only pass what we need into the component.